src/app/features/languageSlice.ts

```diff
@@ -19,6 +21,8 @@ const languageSlice = createSlice({
             i18n.changeLanguage(action.payload).catch((error) => {
                 console.error('Something went wrong loading', error);
```

Comment:
would be great to show some message to user (unknown error or something similar) for all catch blocks